src/components/tag/_tag.scss

```diff
@@ -66,3 +66,90 @@
     }
   }
 }
+
+@mixin tags--x {
+  .#{$prefix}--tag {
+    @include font-family;
+    @include typescale('omega');
+    display: inline-flex;
+    align-items: center;
+    padding: 0 $spacing-xs;
+    height: 1.5rem;
+    margin: $spacing-2xs;
+    border-radius: rem(15px);
+
+    &:not(:first-child) {
+      margin-left: 0;
+    }
+
+    &:focus {
+      outline: none;
+      box-shadow: inset 0 0 0 2px $ibm-colors__blue--60;
```

Comment:
Can any actions be taken on a tag? Are they clickable elements? If so, we should have the `li` wrap `a` or `button` elements, or we should remove the focus state.

Reply:
There are more actions available that aren't currently applied (alison told me to just do the basic for now). But it will be selectable and it can also accept and close button if you want users to be able to close them.